3-semantic-html/basic.html

```diff
@@ -1,11 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Basic semantic in HTML</title>
+    </head>
     
-</body>
+    <body>
+        <header>
+            <nav></nav>
+        </header>
+
+        <main>
+            <article>
+                <header>
+                    <h1>VIM editor</h1>
+                </header>
+                <section>
+                    <p>
+                        Vim is a clon with additions of "VI" the editor made for Unix OS, this was released to public in 1991. Vim is designed for use both from a CLI(<mark>command line interface</mark>) as and a standalone application in a graphic user interface.
+                        <blockquote>
+                            Vim's interface is not based on menus or icons but on commands given in a text user interface.
+                        </blockquote>
+                        Vim has 12 modes, here i mention the basic ones:
+                        <ul>
+                            <li>Normal</li>
+                            <li>Visual</li>
+                            <li>Select</li>
+                            <li>Insert</li>
+                            <li>Command Line</li>
+                            <li>Ex</li>
+                        </ul>
+                    </p>
+                </section>
+                <footer>
+                    <p>
+                        Vim has customization, more features, scripts and much more but you can read about this in the web ...
+                        <br>
+                        References : <cite>Wikipedia</cite> <br>
+                        <a href="https://en.wikipedia.org/wiki/Vim_(text_editor)" target="blank">Vim on wikipedia !</a>
+                        <br> <br> <br>
+                        This site was created 
+                        <time datetime="2020-04-20 20:42">at 20:42</time>
+                    </p>
+                </footer>
+            </article>
+        </main>
+
+        <aside>
+
+        </aside>
+
+        <footer>
+            <nav></nav>
+        </footer>
```

Comment:
Please add content inside the `<footer>`. Same with the `<nav>`. I know that in the image of the example it's empty but it's because we want you fill the content.